src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -315,15 +315,15 @@ module Types = struct
       ~values:
         [enum_value "PLUS" ~value:Sgn.Pos; enum_value "MINUS" ~value:Sgn.Neg]
 
-  let signed_fee =
-    obj "SignedFee" ~doc:"Signed fee" ~fields:(fun _ ->
+  let signed_amount =
+    obj "SignedAmount" ~doc:"Signed amount" ~fields:(fun _ ->
```

Comment:
it's important we keep the graphql API backwards compatible to not break folks' integrations. Instead of making these changes like this, you can copy and paste these fields/objects and mark the existing ones as `~deprecated` (you can grep around in this file to see how that works)